assets/helpers/page/page.js

```diff
@@ -49,6 +52,15 @@ function analyticsInitialisation(participations: Participations): void {
   ga.setDimension('experience', abTest.getVariantsAsString(participations));
   ga.trackPageview();
 
+  const dimensions:Dimensions = {
+    campaignCodeBusinessUnit: getQueryParameter('CMP_BUNIT') || undefined,
+    campaignCodeTeam: getQueryParameter('CMP_TU') || undefined,
+    experience: abTest.getVariantsAsString(participations),
+  };
+
+  googleTagManager.init();
+  googleTagManager.pushDimensions(dimensions);
```

Comment:
Is it worth passing these dimensions straight to `init`? This would reduce the number of calls, and means you could merge the `pushBasicDimensions` and `pushDimensions` function into one.

Reply:
I prefer to keep it separate since we would need pushDimensions in a place without an init.